src/blog/2024/10/announcement-mqtt-broker.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Announcement: MQTT Service Now Available on FlowFuse"
+subtitle: We are thrilled to announce a significant milestone for FlowFuse, we now offer our very own MQTT service, built-in and ready to use with your node-RED applications.
+description: We are thrilled to announce a significant milestone for FlowFuse, we now offer our very own MQTT service, built-in and ready to use with your node-RED applications.
+date: 2024-10-30
+authors: ["joe-pavitt"]
+image: /blog/2024/10/images/dashboard-1-18-0-blog-tile.png
+tags:
+   - posts
+   - news
+   - flowfuse
+   - node-red
+---
+
+You may have seen in our [recent product update](/blog/2024/10/flowfuse-release-2-10) that we have release our very own MQTT service, built-in and ready to use alongside your Node-RED applications. We are always engaging with users and prospective customers and this has consistently been a request, and so we are delighted to announce that this is now live on FlowFuse Cloud for our Team and Enterprise teams.
+
+<!--more-->
+
+The MQTT Service is available now on FlowFuse Cloud. FlowFuse permits you to setup your own secure clients to begin publishing and subscribing to your own topics.
+
+You can now use FlowFuse to manage your own MQTT Clients alongside your Node-RED instances, making it easier to build full-stack, event-driven applications within FlowFuse.
+
```

Comment:
Can we add a little bit more information here on the role MQTT plays in industrial applications/deployments. A use case or two will be helpful to understand more on why we added it to the platform.

Reply:
@joepavitt I will review the doc structure and see what fits the narrative.

Reply:
```suggestion

## Use Cases

* Connect your MQTT enabled PLCs (like Omron N Series, Siemens S7, etc) to your Node-RED instance to open up such possibilities like:
    * Data collection and analysis like Predictive Maintenance, OEE, Condition Based Monitoring. 
    * Triggering actions like sending Emails or alerting your engineers about an event.
    * Realtime production monitoring of your facility.
* Make use of modern IIoT devices on legacy systems by bridging the gap with MQTT.
* Connect disparate systems together where Node-RED and FlowFuse act as the central hub for data processing and routing. This gives you the advantage of transforming data on the fly, applying extra contextual data from other systems, apply routing rules, and much more.

```

@joepavitt @Hasmin-AC - for your consideration

Reply:
Reads good to me - @Hasmin-AC?